source/views/building-hours/report/editor.tsx

```diff
@@ -65,13 +65,38 @@ export function BuildingHoursScheduleEditorView(): JSX.Element {
 
 	let {open, close} = parseHours(set, moment())
 
+	let [summary, setSummary] = React.useState('')
+
+	let buildSummary = React.useCallback(
+		(schedule: SingleBuildingScheduleType): string => {
+			if (!schedule.days.length) {
+				return 'Select a day to get started.'
+			}
+
+			return summarizeDaysAndHours(schedule)
+		},
+		[],
+	)
+
+	React.useEffect(() => {
+		setSummary(buildSummary(set))
+	}, [buildSummary, set])
+
 	return (
 		<ScrollView>
 			<TableView>
-				<Section>
+				<Section footer={summary}>
 					<WeekToggles days={set.days} onChangeDays={onChangeDays} />
-					<DatePickerCell date={open} onChange={onChangeOpen} title="Open" />
-					<DatePickerCell date={close} onChange={onChangeClose} title="Close" />
+					<Cell
+						cellAccessoryView={
+							<Row style={styles.datePickerGroup}>
+								<DatePickerAccessory date={open} onChange={onChangeOpen} />
+								<Text style={styles.datePickerDash}> — </Text>
```

Comment:
> there's extra space after the — - [start] — [end] is how it looks to me. Can we replace the spaces with margin?

The extra spacing is a little trickier. The date pickers grow and shrink in size but I can try margin before saying otherwise. For reference here is how they can change in size...

Do we have other ideas on how to get them to position correctly?

~ | ~ 
--|--
<img width="389" alt="Screen Shot 2022-06-26 at 4 11 07 PM" src="https://user-images.githubusercontent.com/5240843/175837532-b1ba1b8f-67ca-4cce-9e6e-821c8df2e066.png"> | <img width="389" alt="Screen Shot 2022-06-26 at 4 12 07 PM" src="https://user-images.githubusercontent.com/5240843/175837534-1b8aa072-28cd-434b-ae9b-d9e1879f41e1.png">

Reply:
This has improved significantly from switching iOS' `display` to inline and away from compact!

~ | ~
--|--
<img width="389" alt="Screen Shot 2022-06-26 at 4 33 52 PM" src="https://user-images.githubusercontent.com/5240843/175838167-a031da49-9f07-4313-ac23-14acdaef38da.png"> | <img width="389" alt="Screen Shot 2022-06-26 at 4 34 07 PM" src="https://user-images.githubusercontent.com/5240843/175838168-d3f4f159-94db-4fee-861e-c936b365402f.png">